### PR TITLE
Custom limits

### PR DESCRIPTION
**Pull request by KodieGlosserIBM**

### CODE REVIEW

**Comment by KodieGlosserIBM** [comment]
Related issue: https://github.com/openshift/hypershift-toolkit/issues/88

File: pkg/api/types.go [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
+	OpenVPNClientResources              []ResourceRequests `json:"openVPNClientResources"`
+	OpenVPNServerResources              []ResourceRequests `json:"openVPNServerResources"`
 }
 
 type NamedCert struct {
 	NamedCertPrefix string `json:"namedCertPrefix"`
 	NamedCertDomain string `json:"namedCertDomain"`
 }
+
+type ResourceRequests struct {
```
Comment (csrwng) [inline pkg/api/types.go]: nit: s/ResourceRequests/ResourceRequirements/ as in API (https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2087)

File: pkg/api/types.go [hunk 2]
```diff
@@ -1,38 +1,65 @@
...
+	Limits   []Limits   `json:"limits"`
+	Requests []Requests `json:"requests"`
+}
+
+type Limits struct {
```
Comment (csrwng) [inline pkg/api/types.go]: s/Limits/ResourceLimit/

File: pkg/api/types.go [hunk 3]
```diff
@@ -1,38 +1,65 @@
...
+	CPU    string `json:"cpu"`
+	Memory string `json:"memory"`
+}
+
+type Requests struct {
```
Comment (csrwng) [inline pkg/api/types.go]: s/Requests/ResourceRequest/

**Review by csrwng — COMMENTED** [review]
@KodieGlosserIBM looks good to me, just some naming nits

**Pull request merged**